SCF/SCF-3/Calculator.java

```diff
@@ -0,0 +1,153 @@
+
+public class Calculator implements GeneralCalculator {
+	
+	/*
+	 * method to add two hexadecimal numbers 
+	 * @param firstHexaValue and secondHexaValue as String
+	 * @returns sum of two hexadecimal numbers as string
+	 */
+	public String addHexaNum(String firstHexaValue, String secondHexaValue){
+		Conversion convert = new Conversion();
+		int firstDecValue = convert.hexaToDec(firstHexaValue);

```

Comment:
Variables which are used many times in different methods should be declared as class members.